plugins/RocketsPlugin/RocketsPlugin.cpp

```diff
@@ -999,6 +1005,96 @@ class RocketsPlugin::Impl
                       buildJsonRpcSchemaRequest<ModelDescriptor, bool>(desc));
     }
 
+    void _handleGetModelProperties()
+    {
+        const RpcParameterDescription desc{
+            METHOD_GET_MODEL_PROPERTIES,
+            "Get the properties of the given model", "ids",
```

Comment:
I thought that get model properties is for a single model:
```
RPC get-model-properties {id: 'some model id'}
```

Reply:
Yes, it is. Have to fix this description.

Reply:
Sounds good.